conf/distro/hm-generic-console.conf

```diff
@@ -13,4 +13,9 @@ VIRTUAL-RUNTIME_init_manager = "systemd"
 PREFERRED_PROVIDER_udev = "systemd"
 PREFERRED_PROVIDER_udev-utils = "systemd"
 
-EXTRA_IMAGE_FEATURES += "serial-autologin-root"
\ No newline at end of file
+EXTRA_IMAGE_FEATURES += "serial-autologin-root allow-root-login"
```

Comment:
Does this require a root password over serial or will it grant access without entering a password over that interface?

Reply:
still password over serial when i tested it. ssh trough eth0, worked and the same required password.